testsuite/run_forge.sh

```diff
@@ -13,8 +13,8 @@
 pwd | grep -qE 'aptos-core$' || (echo "Please run from aptos-core root directory" && exit 1)
 
 # for calculating regression
-TPS_THRESHOLD=4000
-P99_LATENCY_MS_THRESHOLD=5000
+TPS_THRESHOLD=5000
+P99_LATENCY_MS_THRESHOLD=6000
```

Comment:
these thresholds will actually not block code atm. for that, you'll need to do like  `FORGE_EXIT_CODE=<nonzero>` where we're actually checking the test metrics, e.g.: https://github.com/aptos-labs/aptos-core/pull/2061/files#diff-91b1429f94693e7be52e4ea27f6015b8983411296ba4e5c0a37fe71655823562R168